src/main/java/com/dataguise/saas/service/JWTService.java

```diff
@@ -0,0 +1,56 @@
+package com.dataguise.saas.service;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.apache.log4j.Logger;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Date;
+
+import static java.util.Collections.emptyList;
+
+public class JWTService {
+
+    static final Logger logger = Logger.getLogger(JWTService.class);
+    static final long EXPIRATIONTIME = 300000; // 5minutes, can be reduced after initial tests
+    static final String SECRET = "k5v2nEkDNROaTYK7k96g6ZKofmLsFMOX"; // This should be externalized, but depends on other encryption infrastructure and would most probably need DgController support
+    static final String TOKEN_PREFIX = "Bearer";
+    static final String HEADER_STRING = "Authorization";
+
+    public static void addAuthentication(HttpServletResponse res, String username) throws IOException {
+        String JWT = Jwts.builder()
```

Comment:
**Reporter**: PMD    
    **Rule**: LocalVariableNamingConventions
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/service/JWTService.java L25
    
    The local variable name &#39;JWT&#39; doesn&#39;t match &#39;[a-z][a-zA-Z0-9]*&#39;&#10;&#10;&#10;Code Style https://pmd.github.io/pmd/pmd_rules_java_codestyle.html#localvariablenamingconventions
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1933160064>*